### ISSUE TITLE
Add newsletters to website

### ISSUE THREAD

**Issue by ColeDCrawford** [dh-tech/dh-tech.github.io#82]

**Comment by ColeDCrawford** [comment]
Great question. I have no recollection ... several items came up when we were reviewing the website before switching over to the new build and this was one of them. If they are being added to the website when they get sent out to the mailing lists then I think this can be closed.

**Comment by rlskoeser** [comment]
I agree  - I've been meaning to ask if this issue should be closed because the work has been completed now.

**Comment by rlskoeser** [comment]
Closing since newsletters are now being added to the website regularly.
